rewrite-maven/src/main/java/org/openrewrite/maven/search/ParentPomInsight.java

```diff
@@ -52,6 +58,12 @@ public class ParentPomInsight extends Recipe {
     @Nullable
     String version;
 
+    @Option(displayName = "Recursive",
+            description = "Whether to search recursively through the parents. True by default.",
+            required = false)
+    @Nullable
+    Boolean recursive;
+
```

Comment:
Yes, since this is breaking change I though it does not cost much to still support it. Maybe some people would search for direct usage of a specific parent?